python/modules/connection.py

```diff
@@ -18,6 +18,8 @@
 from . import Module
 from message import Message
 from helpers import serialize_bytes_json, bytes_to_str, str_to_bytes
+from test_suite.tests.connection import Connection as ConnectionValidator
```

Comment:
I don't think production functionality should be importing from a test suite; dependencies should flow the other direction.

Reply:
This is a reference agent so not that bad but i pointed this exact issue to @dbluhm yesterday over email that we need a separate utilities package that both the reference agent and test-suite should use. I can address that in this PR too but then this would be a bigger one.

Reply:
Okay, I am okay with merging if we are agreed that we will make that second change quickly.

Reply:
I can make that change part of PR #104

Reply:
Given the tenuous nature of the python ref agent at this point, I'm reluctant to have any imports between the ref agent and test suite.

Thoughts?

Reply:
I'd rather create a python-framework path in the repo that they both import from. We don't yet have a plan for a python framework (just a python idiomatic library) and this could be the beginning of that.

Reply:
Alright, i will move the common code in a different package accessible to both the ref agent and test-suite in PR #104

Reply:
I have moved non testing logic to reference agent in PR #104

Reply:
I'm happy with the uncoupling of the code in that PR, but the package import renaming issue remains.